client/my-sites/site-settings/sso.jsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import JetpackModuleToggle from 'my-sites/site-settings/jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import CompactFormToggle from 'components/forms/form-toggle/compact';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { isJetpackModuleActive } from 'state/selectors';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+
+const Sso = ( {
+	fields,
+	handleToggle,
+	isRequestingSettings,
+	isSavingSettings,
+	selectedSiteId,
+	ssoModuleActive,
+	translate
+} ) => {
+	return (
+		<div>
+			<Card className="sso__card site-settings__security-settings">
+				<FormFieldset>
+					<div className="sso__info-link-container site-settings__info-link-container">
+						<InfoPopover position={ 'left' }>
+							<ExternalLink href={ 'https://jetpack.com/support/sso' } target="_blank">
+								{ translate( 'Learn more about WordPress.com Secure Sign On' ) }
+							</ExternalLink>
+						</InfoPopover>
+					</div>
+
+					<JetpackModuleToggle
+						siteId={ selectedSiteId }
+						moduleSlug="sso"
+						label={ translate( 'Allow log-in using WordPress.com accounts.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **11** times:
`translate( 'Allow your users to log in using their WordPress.com accounts.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).